index.js

```diff
@@ -91,453 +76,471 @@ function serveIndex(root, options) {
     throw new TypeError('serveIndex() root path required');
   }
 
+  var pathLib = opts.path || nativePathLib;
+  var normalize = pathLib.normalize
+  , extname = pathLib.extname
+  , sep = pathLib.sep
+  , join = pathLib.join
+  , resolve = pathLib.resolve;
+
+  // default Stylesheet
+  var defaultStylesheet = join(__dirname, 'public', 'style.css');
+  // default Template
+  var defaultTemplate = join(__dirname, 'public', 'directory.html');
```

Comment:
these two are for the module to find its own static file from disk, therefore, they should use native path.join.

Reply:
corrected and moved it back to global scope.